net.sourceforge.lgames.LBreakoutHD.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
upstream has a desktop file, so why adding one here?

https://sourceforge.net/p/lgames/code/HEAD/tree/trunk/lbreakouthd/lbreakouthd.desktop.in

Reply:
That one is very basic, lacking keywords and categories that would make the game more easy to find on flathub.org

Reply:
1. it doesn't changed the search on flathub. only the appdata matters
2. `desktop-file-edit` can be used in the post-install to add things to upstream .desktop file as needed.

Reply:
> 1. it doesn't changed the search on flathub. only the appdata matters

I think you're wrong, the keywords defined in the .desktop file are connected to the search on flathub, i.e. you can search any app on flathub.org by the keywords in the app .desktop file.

Reply:
even if I am, 2. still stands. And then you can submit a patch upstream.

Reply:
Submitted -> https://sourceforge.net/p/lgames/patches/29/